pkg/cmd/operator/operator_test.go

```diff
@@ -1,24 +1,33 @@
 package operator_test
```

Comment:
The tests dont really do much for the skip ns creation, I ran them to check the output of the command.
Will see how to make these tests more effective soon ...